product_equivalence/README.rst

```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product equivalence
+===================
+
+This module adds equivalent product link in product
+
+Usage
+=====
+
+This module adds a page with field "equivalent product" and compatible products (no business logic for this field)
+If product have equivalent it will be replaced in delivery order
+
+Contributors
+------------
+
+* "Camptocamp, "
+* "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+* "Odoo Community Association (OCA)",
```

Comment:
Here please list the following contributors:

     *  Nicolas Bessi <nicolas.bessi@camptocamp.com>
     * Yannick Vaucher <yannick.vaucher@camptocamp.com>
     * Guewen Baconnier <guewen.baconnier@camptocamp.com>
     * Anar Baghirli